Create: issue templates (#339)
XTLS/Xray-core: .github/ISSUE_TEMPLATE/question-en.md

---
name: Question
about: Question of Xray.
title: ''
labels: ''
assignees: ''

---

> <!-- Thanks for your supporting！

Reviewer: ```suggestion
<!-- Thanks for your support！
```

Reviewer: sup·port ★ / səˈpɔːt   ; NAmE səˈpɔːrt   / verb , noun
Support is also a noun.
The prime minister gave his full support to the government's reforms.
首相对政府改革予以全力支持。
They are prepared to resort to violence in support of their beliefs.
他们准备诉诸暴力来捍卫自己的信仰。

> 1. This isn't a proper place to aks question, otherwise your issue may be closed.

Reviewer: ```suggestion
1. This is **NOT** a proper place to ask a question, otherwise your issue may be closed.
```

> 2. Please check the existing issues, discussions and documentation in detail. You may find the answers you need there.

Reviewer 2: ```suggestion
2. Please check the existing issues, discussions and documentation in detail first. You may find the answers you want before you ask the question.
```
